docs/commands/secrets.md

```diff
@@ -187,56 +187,71 @@ To ensure that your command’s arguments and options are passed correctly you m
 $ torus run -e production -s www -- node ./bin/www --app api
 ```
 
-## ls
-###### Added [v0.13.0](https://github.com/manifoldco/torus-cli/blob/master/CHANGELOG.md)
-
-`torus ls [path]` allows a user to explore the objects and values set within their organization.
+## list
+###### Added [v0.28.0](https://github.com/manifoldco/torus-cli/blob/master/CHANGELOG.md)
 
-Each level within the organization can be inspected by changing the segments supplied in the path. Wildcards cannot be used for the organization or project segments of the [path](../concepts/path.md).
+`torus list [name]...` allows a user to explore the objects and values set within their organization. Users can find the location of a particular secret by passing its name as an argument, or view all secrets within different branches of their organization.
 
-Path is required, and does not support context.
+Each level within the organization can be inspected by changing the options supplied to the command.
```

Comment:
If you were to take my suggestion, I think this sentence would be redundant.

Reply:
Yep, you're right.